File: android/src/main/kotlin/com/example/design_engine/layer2_core/light/ETertiaryContainerLight.kt
package com.example.design_engine.layer2_core.light
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object ETertiaryContainerLight {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyl35
            AppTheme.STANDARD -> R.color.cstl35
            AppTheme.BLUE -> R.color.cbl35
            AppTheme.GREEN -> R.color.cgl35
            AppTheme.RED -> R.color.crl35
            AppTheme.NOTHING -> R.color.cnol35
        }
    }
}

File: lib/layer2_core/light/e_tertiary_container_light.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/light/system.dart';
import '../../layer1_raw/light/standard.dart';
import '../../layer1_raw/light/blue.dart';
import '../../layer1_raw/light/green.dart';
import '../../layer1_raw/light/red.dart';
import '../../layer1_raw/light/nothing.dart';

class ETertiaryContainerLight {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyl35;
      case 'standard': return cstl35;
      case 'blue': return cbl35;
      case 'green': return cgl35;
      case 'red': return crl35;
      case 'nothing': return cnol35;
      default: return cstl35;
    }
  }
}
